### PR TITLE
Karolina navigation

### PR DESCRIPTION
**Pull request by moharso**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+        <h1>Join us at one of our events.</h1>
+        <p>
+          Become part of a community of productivity experts with virtual and
+          in-person events to learn strategies for work, make new friends, and
+          more!
+        </p>
+      </div>
+      <div
+        class="background-img"
+        style="
```
Comment (ZymanteL) [inline index.html]: Maybe style could be in css file.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  letter-spacing: -0.2rem;
+}
+
+.description p {
+  padding: 0.5rem;
+  font-size: 1.6rem;
+  width: 70vh;
+  text-align: center;
+  letter-spacing: 0.02rem;
+}
```
Comment (ZymanteL) [inline styles.css:131]: You should add some @media.

Comment (moharso) [reply styles.css:131]: Working on that :)

File: index.html (hunk 1 above)
Comment (moharso) [reply index.html]: Will do that, thank you!

**Review by ZymanteL — APPROVED** [review]
Looks fine.

**Pull request merged**